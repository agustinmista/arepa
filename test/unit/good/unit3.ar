(module unit3
  (val main
    (let ((u  (print_int 10))
          (u' (trace_unit u)))
      (print_unit u')))

  (fun print_int  (x) (__prim_print_int__ x))
  (fun print_unit (x) (__prim_print_unit__ x))
  (fun trace_unit (x) (__prim_trace_unit__ x))
)