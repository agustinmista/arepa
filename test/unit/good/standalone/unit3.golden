10
<unit(0)>
unit(0)
